src/components/article-list.jsx

```diff
@@ -1,7 +1,36 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils.js/API";
+import ArticleCard from "./article-card";
+
+
+
+
 function ArticleList() {
+
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] =useState(true)
+    
+    useEffect(() => {
+        setIsLoading(true);
+        getArticles().then((articleData)=> {
+            
+          setArticles(articleData)
+          
+         
+          setIsLoading(false)
+        })
+      }, []) 
+
+    
+      
     return (
     <section className="article-list">
-        <h3>ArticleList</h3>
+        <h3>articlessss</h3>
+        <ul>
+        <ArticleCard 
+                articles={articles}
+                />
+        </ul>
```

Comment:
You should to use your isLoading state to let the user know it's loading - right now you've got the state but you're not using it to show anything. You'll want some kind of conditional render here to show a message while the data is still being fetched behind the scenes